contracts/interfaces/ISDUtilityPool.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.16;
+
+struct UserData {
+    uint256 totalInterestSD;
+    uint256 totalCollateralInSD;
+    uint256 healthFactor;
+    uint256 lockedEth;
+}
+
+struct Config {
+    uint256 liquidationThreshold;
+    uint256 liquidationBonus;
+    uint256 ltv;
+}
+
+struct OperatorLiquidation {
```

Comment:
Some documentation explaining what these fields are for all the structs is helpful.